src/sidebar/openInZeplin/command/OpenInZeplinOnDoubleClickCommand.ts

```diff
@@ -0,0 +1,28 @@
+import Command from "../../../common/vscode/command/Command";
+import { openInZeplin } from "../flow/openInZeplinFlow";
+import ZeplinUriProvider from "../model/ZeplinUriProvider";
+
+const DOUBLE_CLICK_DURATION = 300;
```

Comment:
Here is a great comment about the double click experience https://ux.stackexchange.com/a/63412 😂 

I am not sure are there any default timeframe in this kind of extension/webapp experiences. But looks like windows has a default of 500ms double-click timeframe between the clicks. I believe macos has a very similar value.

I guess it would be better and safer to follow the experience users have gotten used to. WDYT?

Reply:
😮 whoaa, community tested answer. I do not have authority to overrule that. I will fix it asap. 👍